src/pkg/scan/report/base_manager.go

```diff
@@ -237,3 +239,48 @@ func (bm *basicManager) DeleteByDigests(digests ...string) error {
 
 	return err
 }
+
+// GetStats ...
+func (bm *basicManager) GetStats(requester string) (*all.Stats, error) {
+	if len(requester) == 0 {
+		return nil, errors.New("empty requester")
+	}
+
+	m, err := scan.GetScanStats(requester)
+	if err != nil {
+		return nil, errors.Wrap(err, "report manager: get stats")
+	}
+
+	sts := &all.Stats{
+		Metrics: make(all.StatusMetrics),
+	}
+
+	for k, v := range m {
+		// Increase the total metrics
+		sts.Total += v
+
+		cc := job.Status(k).Code()
+		// Increase the completed metrics
+		if cc == -1 || cc == job.SuccessStatus.Code() {
```

Comment:
add comments to explain `-1` and job.SuccessStatus.Code() is actually `Stopped or Error or  Success`.